lib/loader/index.js

```diff
@@ -135,6 +186,16 @@ module.exports = {
       var firstScript = getFirstScript();
 
       firstScript.parentNode.insertBefore(script, firstScript);
+
+      _loadedUrls[url] = true;
+
+      if (options.namespaceName) {
+        namespace.loadedUrls = _loadedUrls;
```

Comment:
What @sirugh said - use of registerProperty is just a way to ensure that when there are conflicts, there will be a clear exception rather than confusion as two apps edit each other's data haphazardly.

I don't think there is any intent of having app2 read the data used by app1.